dgraphtest/local_cluster.go

```diff
@@ -42,8 +44,8 @@ type cnet struct {
 }
 
 type LocalCluster struct {
-	conf ClusterConfig
-
+	conf       ClusterConfig
+	tempBinDir string
```

Comment:
just binDir is not fine?

Reply:
We are using binDir to store all versions binaries and tempDir to mount specific version into container.